jest.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     "^uuid$": "<rootDir>/node_modules/uuid/dist/index.js",
   },
   modulePaths: ["<rootDir>/src"],
-  setupFiles: ["whatwg-fetch"],
+  setupFiles: ["whatwg-fetch", "./jest.setup.ts"],
```

Comment:
if we're using a Jest setup file now, could we import `whatwg-fetch` there instead? The reason `whatwg-fetch` was on its own was because it avoided having a one line file with an import statement